test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -353,7 +393,7 @@ class CloudFoundryDeployTest extends BasePiperTest {
             cfManifest: 'test.yml'
         ])
 
-        assertThat(shellRule.shell, not(hasItem(containsString("cf stop testAppName-old"))))
+        assertThat(shellRule.shell, not(hasItem(containsString("cf stop testAppName-old &> 1-cfStopOutput.txt"))))
```

Comment:
Same here: Why is it 1-cfStopOutput.txt?

Reply:
done